packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -192,9 +192,17 @@ class PerformanceController
   final _displayRefreshRate = ValueNotifier<double>(defaultRefreshRate);
 
   Future<void> selectFrame(FlutterFrame frame) async {
```

Comment:
method name is now misleading. Maybe it is now "clickFrame" or "toggleFrame"

Reply:
changed